EULA.md

```diff
@@ -0,0 +1,291 @@
+    the Services, or shared areas of the Services you haven't been given
+    access to;
+
+4) willfully interfere with or disrupt any user, host, or network, for
+    example by sending a virus, overloading, flooding, spamming, or
+    mail-bombing any part of the Services;
+
+5) access, search, or create accounts for the Services by any means
+    other than the Provider’s publicly supported interfaces (for
+    example, "scraping" or creating accounts in bulk);
+
+6) send unsolicited communications, promotions or advertisements, or
+    spam;
+
+7) send altered, deceptive or false source-identifying information,
+    including "spoofing" or "phishing";
+
+8) promote or advertise products or services without appropriate
+    authorization;
+
+9) circumvent quotas, including computer, networking and storage
+    limits;
+
+10) sell the Services unless specifically authorized to do so;
+
+11) publish or share materials that are unlawfully pornographic or
+    indecent, or that contain extreme acts of violence;
+
+12) advocate bigotry or hatred against any person or group of people
+    based on their race, religion, ethnicity, sex, gender identity,
+    sexual preference, disability, or impairment;
+
+13) violate the law in any way, including storing, publishing or sharing
+    material that is fraudulent, defamatory, or misleading;
+
+14) violate the privacy or infringe the rights (including those rights
+    granted under trademark and copyright law) of others;
+
+15) use any component, function, or other facility of the Services to
+    store, access, transmit, or share any “personal information” of
+    individuals (as such term is defined in Section 1 of M.G.L. c. 93H);
+
+16) If you are a covered entity, business associate, or representative
+    of a covered entity or business associate (as those terms are
+    defined by the Health Insurance Portability and Accountability Act
+    of 1996 (“HIPAA”) at 45 C.F.R. § 160.103), use any component,
+    function, or other facility of the Services to store, access,
+    transmit, or share any “protected health information” (as such term
+    is defined at 45 C.F.R. § 160.103) or use the Services in any manner
+    that would make the Provider your or any third party’s business
+    associate;
+
+17) access the Services from sanctioned countries, per the list kept by
+    the Bureau of Industry and Security of the U.S. Department of
+    Commerce; or
+
+18) use or access the Services in any way that would make the Provider
+    subject to the law of any jurisdiction outside of the United States
+    requiring the Provider to take affirmative steps in order to store,
+    access, use, transfer, or otherwise process particular User Content
+    that includes personal information of individuals or other sensitive
+    information legally (e.g., the EU’s General Data Protection
+    Regulation, China’s Personal Information Protection Law, the U.S.’s
+    HIPAA).
+
+19. **Removal of Content**
```

Comment:
This looks like an error (it should be a section heading, not the 19th item of the preceding list).